TrafficCapture/dockerSolution/src/main/docker/docker-compose.yml

```diff
@@ -78,6 +78,26 @@ services:
       - migrations
     ports:
       - "29200:9200"
+#  opensearch-benchmarks:
+#    image: 'migrations/open_search_benchmark:latest'
+#    networks:
+#      - migrations
+#    depends_on:
+#      - captureproxy
+#    command:
+#      - /bin/sh
+#      - -c
+#      - >
+#        echo "Running opensearch-benchmark w/ 'geonames' workload..." &&
+#        opensearch-benchmark execute-test --distribution-version=1.0.0 --target-host=https://captureproxy:9200 --workload=geonames --pipeline=benchmark-only --test-mode --kill-running-processes --workload-params "target_throughput:0.5,bulk_size:10,bulk_indexing_clients:1,search_clients:1"  --client-options="use_ssl:true,verify_certs:false" &&
+#        echo "Running opensearch-benchmark w/ 'http_logs' workload..." &&
+#        opensearch-benchmark execute-test --distribution-version=1.0.0 --target-host=https://captureproxy:9200 --workload=http_logs --pipeline=benchmark-only --test-mode --kill-running-processes --workload-params "target_throughput:0.5,bulk_size:10,bulk_indexing_clients:1,search_clients:1"  --client-options="use_ssl:true,verify_certs:false" &&
+#        echo "Running opensearch-benchmark w/ 'nested' workload..." &&
+#        opensearch-benchmark execute-test --distribution-version=1.0.0 --target-host=https://captureproxy:9200 --workload=nested --pipeline=benchmark-only --test-mode --kill-running-processes --workload-params "target_throughput:0.5,bulk_size:10,bulk_indexing_clients:1,search_clients:1"  --client-options="use_ssl:true,verify_certs:false" &&
+#        echo "Running opensearch-benchmark w/ 'nyc_taxis' workload..." &&
+#        opensearch-benchmark execute-test --distribution-version=1.0.0 --target-host=https://captureproxy:9200 --workload=nyc_taxis --pipeline=benchmark-only --test-mode --kill-running-processes --workload-params "target_throughput:0.5,bulk_size:10,bulk_indexing_clients:1,search_clients:1"  --client-options="use_ssl:true,verify_certs:false" &&
+#        sleep 30 &&
+#        curl --insecure https://captureproxy:9200/
```

Comment:
To compensate for the shortcoming that the proxy solution doesn't yet send close events.  This will add one more captured event far enough past all of the previously seen packets that the replayer has seen, causing all previous packets to be flushed.